programs/programs/cross_white_label/ccdf/base.py

```
@@ -0,0 +1,29 @@
+        dependency.member.CcdfReasonCareEligibleDependency,
+        dependency.member.EmploymentIncomeDependency,
+        dependency.member.SelfEmploymentIncomeDependency,
+        dependency.member.PensionIncomeDependency,
+        dependency.member.InvestmentIncomeDependency,
+        dependency.member.RentalIncomeDependency,
+        dependency.member.MiscellaneousIncomeDependency,
+    ]
+    pe_outputs = [dependency.member.Ccdf]
+
+    def child_care_cost(self, member: HouseholdMember) -> int:
+        raise NotImplemented("Please define the 'child_care_cost' method")
```

Ruff (0.16.1)

[error] `raise NotImplemented` should be `raise NotImplementedError`

Use `raise NotImplementedError`

(F901)